src/app/component/CategoryPagination/CategoryPagination.component.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { CategoryTreeType } from 'Type/Category';
+import { generateQuery } from 'Util/Url';
+import './CategoryPagination.style';
+
+class CategoryPagination extends Component {
+    renderPageLink(pageNumber, isCurrent) {
+        const {
+            category: { url_path }, getPage, location, history
+        } = this.props;
+
+        const page = pageNumber !== 1 ? pageNumber : '';
+        const search = generateQuery({ page }, location, history);
+
+        return (
+            <Link
+              to={ {
+                  pathname: `/category/${ url_path }`,
+                  search
+              } }
+              aria-label={ __('Page %s', pageNumber) }
+              className={ isCurrent ? 'active' : '' }
```

Comment:
Let's give link a proper className so we can style it without increasing selector depth.